docs/static/templates/multiple-tasks-template.md

```diff
@@ -32,10 +32,10 @@ Replace any text in _italics_ with content for your procedure.
 
 _Goal of procedure_:
 
-1. Configure service
-1. Set up the environment
-1. Optional: Install libraries
-1. Verify the installation
+1. [Configure service](#configure-service)
```

Comment:
@knylander-grafana , I believe there was a reason why we decided not to include in-page links. Was it a maintenance consideration? And should we reconsider? Adding links is more beneficial for the user, but we need to weigh that against maintenance and potentially breaking links when headings change.

Reply:
> @knylander-grafana , I believe there was a reason why we decided not to include in-page links. Was it a maintenance consideration? And should we reconsider? Adding links is more beneficial for the user, but we need to weigh that against maintenance and potentially breaking links when headings change.

https://github.com/grafana/writers-toolkit/blob/main/docs/sources/writing-guide/useful-link-text/index.md#quantity

Reply:
@AnneB-SEO2, what is your take on adding these types of links within the same document?

Reply:
@jdbaldry, could we use a `{{< section >}}` or some other method here so these are auto-generated? (Also, if so, could we say "Do not include" for a given section?)

Reply:
> @jdbaldry, could we use a `{{< section >}}` or some other method here so these are auto-generated? (Also, if so, could we say "Do not include" for a given section?)

Oh, but we would need to number them...

Reply:
The `section` shortcode is more for describing the child pages rather than anchors within the page. I don't see why we couldn't have another shortcode that works for anchors. For both shortcodes, it would be possible to include a parameter to switch between unordered and ordered lists.

I'll do it for the `section` shortcode today since it won't take long.